llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -1330,6 +1330,13 @@ class VPInstruction : public VPRecipeWithIRFlags {
     assert(Opcode == Instruction::Or && "only OR opcodes can be disjoint");
   }
 
+  VPInstruction(VPValue *Ptr, VPValue *Offset, bool InBounds, DebugLoc DL = {},
```

Comment:
Seems a bit odd for a constructor of VPInstruction(VPValue*, VPValue*, bool) to create a PtrAdd. Perhaps having GEPFlagsTy as 3rd parameter instead of bool would be clearer. Should be documented.

Reply:
This is part of  #10643